src/styles/styles.js

```diff
@@ -266,6 +266,11 @@ const styles = {
         borderWidth: 0,
     },
 
+    buttonDisabledBG: {
```

Comment:
Sorry, last quick comment here - I think this can be simply called `buttonDisabled` - no need for the extra `BG` here.

Reply:
Fixed.